classes/options-general.php

```diff
@@ -178,7 +178,7 @@ function print_page() {
 
 				// Messages
 				if ( isset( $_GET['sm-msg'] ) && '1' == $_GET['sm-msg'] ) {
-					echo '<div class="updated fade"><p>' . __( 'SimpleMap settings saved.', 'simplemap' ) . '</p></div>';
+					echo '<div class="updated fade"><p>' . __( 'SimpleMap settings saved.', 'SimpleMap' ) . '</p></div>';
```

Comment:
__[code_sniffer]__
* Spaces must be used for alignment; tabs are not allowed
* Space after opening parenthesis of function call prohibited
* Expected 0 spaces before closing bracket; 1 found
* Line exceeds 120 characters; contains 121 characters

<!-- Comment by [SideCI](https://sideci.com) -->